docs/development/workflow/development_workflow.rst

```diff
@@ -457,9 +457,8 @@ start make a branch to serve as a backup copy of your work::
 
     git branch tmp my-new-feature # make temporary branch--will be deleted later
 
-After altering the history with ``git rebase`` or ``git merge --squash``,
-a normal ``git push`` is prevented, and for both a ``git push
---force`` will be required.
+After altering the history, e.g. with ``git rebase``, a normal ``git push``
+is prevented, and for both a ``git push --force`` will be required.
```

Comment:
haha, you indeed find all inconsistencies. Will fix and push an amend and merge once the docs build passed.